UberSignals.podspec

```diff
@@ -9,10 +9,13 @@ Pod::Spec.new do |s|
   s.source = { :git => 'https://github.com/uber/signals-ios.git', :tag => s.version }
   s.requires_arc = true
 
-  s.ios.deployment_target = '7.0'
+  s.ios.deployment_target = '8.0'
```

Comment:
Reverted this back to 7.0 as integrating via cocoapods and a static library requires the deployment target to be 7.0